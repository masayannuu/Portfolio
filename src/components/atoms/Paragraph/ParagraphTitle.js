import React from 'react'
import styled from 'styled-components'

const StyledTitle = styled.p`
  margin-right: 0.8rem;
  font-feature-settings: 'palt';
  letter-spacing: 0.05em;
`

const ParagrahTitle = ({ className, children }) => (
  <StyledTitle className={className}>{children}</StyledTitle>
)

export default ParagrahTitle
