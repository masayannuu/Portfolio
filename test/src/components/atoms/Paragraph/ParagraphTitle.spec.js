import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ParagraphTitle from '../../../../../src/components/atoms/Paragraph/ParagraphTitle'

describe('Paragraph ParagraphTitle Atoms', () => {
  it('should render ParagraphTitle', () => {
    const wrapper = Enzyme.shallow(<ParagraphTitle>ParagraphTitle</ParagraphTitle>)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
