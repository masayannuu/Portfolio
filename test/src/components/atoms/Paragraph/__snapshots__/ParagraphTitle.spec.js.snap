// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph ParagraphTitle Atoms should render ParagraphTitle 1`] = `
<ParagraphTitle__StyledTitle>
  ParagraphTitle
</ParagraphTitle__StyledTitle>
`;
